src/components/Details/VmDetails/VmDetails.js

```diff
@@ -166,99 +169,92 @@ export class VmDetails extends React.Component {
     );
 
     return (
-      <div className="co-m-pane__body">
-        <h1 className="co-m-pane__heading">
-          Virtual Machine Overview
-          <div>{this.state.editing ? cancelSaveButton : editButton}</div>
-        </h1>
-        {this.state.k8sError && <Alert onDismiss={this.onErrorDismiss}>{this.state.k8sError}</Alert>}
-        <Row>
-          <Col lg={4} md={4} sm={4} xs={4} id="name-description-column">
-            <dl>
-              <dt>Name</dt>
-              <dd>{vm.metadata.name}</dd>
-              <dt>Description</dt>
+      <React.Fragment>
+        {!overview && (
```

Comment:
please can you make variables out of these? at least the first one?

Reply:
variable `header` created from both